### PR TITLE
SERVER-1948 |  password generator script

### PR DESCRIPTION
**Pull request by atulsingh0**
:gear: **Issue**

* Generate the password as required by CircleCI Server installation
* [SERVER-1948](https://circleci.atlassian.net/browse/SERVER-1948)

:white_check_mark: **Fix**

:question: **Tests**

- [ ] Tested Updating Existing Instance
- [X] Installed on new instance

### CODE REVIEW

**Review by ryanwohara — APPROVED** [review]

File: passwords/genPass.sh [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    fi
+}
+
+function gen_password(){
+    env LC_ALL=C tr -dc 'A-Za-z0-9_' < /dev/urandom | head -c "$1"
+}
+
+function sign_enc_keys(){
+    sign=$(sudo docker run circleci/server-keysets:latest generate signing -a stdout)
+    enc=$(sudo docker run circleci/server-keysets:latest generate encryption -a stdout)
```
Comment (ryanwohara) [inline passwords/genPass.sh:20]: Maybe a nit: We only need sudo in one place - here or the execution of the script

Comment (ryanwohara) [reply passwords/genPass.sh:20]: Actually the more I think about it the more we should remove this from the script. If the user needs sudo they can run the entire script as sudo.

Comment (atulsingh0) [reply passwords/genPass.sh:20]: removed the sudo from pipe ( in readme)

File: passwords/generate_password.sh [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    fi
+}
+
+function gen_password(){
+    env LC_ALL=C tr -dc 'A-Za-z0-9_' < /dev/urandom | head -c "$1"
+}
+
+function sign_enc_keys(){
+    sign=$(sudo docker run circleci/server-keysets:latest generate signing -a stdout)
+    enc=$(sudo docker run circleci/server-keysets:latest generate encryption -a stdout)
```
Comment (ryanwohara) [inline passwords/generate_password.sh:20]: Can you remove sudo from these commands? People can sudo bash the entire script when necessary.

Comment (atulsingh0) [reply passwords/generate_password.sh:20]: 👍

**Pull request merged**